coderd/rbac/auth_fake.go

```diff
@@ -0,0 +1,31 @@
+package rbac
```

Comment:
this appears to only be used in coderd_test.go --- any reason it can't live there?

Reply:
There isn't. I deleted the test that used a fake authorizer before. 

I will move this :+1: